Tasks/XcodeV5/xcodeutils.ts

```diff
@@ -205,6 +206,23 @@ export function uploadLogFile(logFile: string) {
     }
 }
 
+// Same signature and behavior as utility-common/telemetry's emitTelemetry, minus the common vars.
+export function emitTelemetry(area: string, feature: string, taskSpecificTelemetry: { [key: string]: any; }): void {
+    try {
+        let agentVersion = tl.getVariable('Agent.Version');
+        if (semver.gte(agentVersion, '2.120.0')) {
+            console.log("##vso[telemetry.publish area=%s;feature=%s]%s",
+                area,
+                feature,
+                JSON.stringify(taskSpecificTelemetry));
+        } else {
+            tl.debug(`Agent version of ( ${agentVersion} ) does not meet minimum reqiurements for telemetry`);
```

Comment:
I think it would be helpful to say, "Agent version is ${agentVersion}.  Version 2.120.0 or higher is needed for telemetry."

Reply:
Done.